feat(content): HR player experience updates 3 (#8329)
endless-sky/endless-sky: data/hai/hai reveal 6 end.txt

		log `The Free Worlds, Republic, and Hai Elders have signed a resolution to ensure safe passage for Hai in human space, on the condition that they register with a customs authority on Mountaintop. Any Hai sent against their will into human space will be recovered alive if possible, regardless of the risk to human soldiers. However, human authorities will not change their approach towards regular human pirates and slavery, citing "unsustainable loss of life."`
		log `Have learned that during the crisis a single ship loaded with kidnapped Hai was misplaced and abandoned. They require rescue and may indicate the presence of a loose end.`
		conversation

> 			`Arriving on Mountaintop, you see the station has seen serious renovations, including a grand ballroom, all manner of diplomatic suites, and a notable lack of movie props. Rather than actors and stage crew, you see many familiar and unfamiliar people in full diplomatic regalia. The Chancellor of the Free Worlds and President of the Republic are in their most respectable suits, Alondo and Xilin have the traditional symbols of office (though Alondo's looks like the Free Worlds have perhaps not finalized their design just yet), and dress uniforms laden with medals abound on both sides. You appear to be a guest of honor, being occasionally called upon to answer only a few idle queries from people seeking clarity on one point or another - and otherwise allowed to mingle with anyone taking a break from their duties. Throughout these preliminary meetings, you notice Remington and Terry are conspicuously absent among the attendees.`

Reviewer: ```suggestion
			`Arriving on Mountaintop, you see the station has seen serious renovations including a grand ballroom, all manner of diplomatic suites, and a notable lack of movie props. Rather than actors and stage crew, you see many familiar and unfamiliar people in full diplomatic regalia, and dress uniforms laden with medals abound on all sides. The Chancellor of the Free Worlds and President of the Republic are in their most respectable suits, while Alondo and Xilin have the traditional symbols of office (though Alondo's looks like the Free Worlds have perhaps not finalized their design just yet). You appear to be a guest of honor, occasionally being called upon to answer idle queries from people seeking clarity on one point or another but otherwise left to your self. As you mingle with those taking a break from their duties throughout these preliminary meetings, you notice Remington and Terry are conspicuously not among the attendees.`
```

This is another place I think the additions are good in their own right, but they further convolute and confuse an already awkward passage.

I don't think the comma there is correct/appropriate, but I'm also not sure nothing is. This may be best as `:` after `including`, but I'll admit I'm not very strong on when those are used - just not really my style. Next change is to switch from `X/Y people, X/Y people, and objects` pattern shift in that sentence, especially since it's broken by the parenthetical aside anyway. Next sentence is unnecessarily complex I think, but also using I don't think that `- and` is used correctly. Last is more fixing of the previous, but that `absent among the attendees` also feels awkward/redundant. An alternative would be to make it just ` are conspicuously absent`.

Author: Adopting your suggestion with the following minor tweaks:
1. your self -> yourself
2. make it just  `are conspicuously absent.`
```suggestion
			`Arriving on Mountaintop, you see the station has seen serious renovations including a grand ballroom, all manner of diplomatic suites, and a notable lack of movie props. Rather than actors and stage crew, you see many familiar and unfamiliar people in full diplomatic regalia, and dress uniforms laden with medals abound on all sides. The Chancellor of the Free Worlds and President of the Republic are in their most respectable suits, while Alondo and Xilin have the traditional symbols of office (though Alondo's looks like the Free Worlds have perhaps not finalized their design just yet). You appear to be a guest of honor, occasionally being called upon to answer idle queries from people seeking clarity on one point or another but otherwise left to yourself. As you mingle with those taking a break from their duties throughout these preliminary meetings, you notice Remington and Terry are conspicuously absent.`
```

			choice
				`	(Ask Xilin about it.)`
				`	(Ignore it.)`
					goto publicbroadcast
			branch presaside
				has patronizedthepresident
			`	Between scheduled sessions, Xilin explains the Syndicate's absence. "The Syndicate is technically part of the Republic and awarded no special diplomatic status. Besides, Terry seems to have disappeared from Syndicate communications, though I won't hazard a guess as to their internal politics."`
				goto publicbroadcast
			label presaside
			`	At one quiet point between structured meetings, during which the peripheral conversations are more focused on the particulars of the Free World's position, you manage to get a hold of Xilin to ask about it. He explains the Syndicate's absence. "The Syndicate is technically part of the Republic and awarded no special diplomatic status. Besides, Terry seems to have disappeared from Syndicate communications, though I won't hazard a guess as to their internal politics."`

> 			`	Perhaps seizing on the opportunity to get a momentary reprieve from more official business, the president himself steps in to fill the gap as soon as Xilin departs. It seems that you acquire a bubble of space around you whenever anyone of note stands next to you and the president exploits this. "You know," he says quietly, while staring out at the many people hustling about at their business, "it's not very often that someone has the gall to patronize me to my face... I usually have to pay people to do that, or at least, to do it constructively anyway."`

Reviewer: ```suggestion
			`	Perhaps seizing on the opportunity to get a momentary reprieve from more official business, the President himself steps in to fill the gap as soon as Xilin departs. It seems that you acquire a bubble of space around you whenever anyone of note stands next to you, and the President exploits this. "You know," he says quietly, while staring out at the many people hustling about at their business, "it's not very often that someone has the gall to patronize me to my face... I usually have to pay people to do that, or at least, to do it constructively anyway."`
```
I think that's title, not role, here, so needs capitalized. Also, am I the only one reading `I usually have to pay people to do that` coming across less as an official job offer and more as a... proposition?

Author: The ambiguity is intentional. ;)

			`	The pause he leaves between sentences is just slightly too short to invite a response. "If you ever decide on a career change, that might be an option for you."`
			`	He exhales through his nose as if chuckling at something humorous. "Fly safe Captain, a lot of people know who you are now."`

> 			`	With that he steps away directly to meet an aide who had been hurrying towards you both, intent on ignoring the bubble. From across the room you catch Xilin with his eyebrows raised, giving you a curious look despite being engaged in his own conversation. Neither of them speak to you again for the remainder of the meetings though.`

Reviewer: ```suggestion
			`	With that he steps away directly to meet an aide who had been hurrying towards you both, intent on ignoring the bubble. From across the room you catch Xilin with his eyebrows raised, giving you a curious look despite being engaged in his own conversation. Neither of them speak to you again for the remainder of the meetings, though.`
```

Author: I personally hate this `<comma> though <end>` construction more than possibly anything else that comes up here. It doesn't play nice with my local english. Is there an alternative construction?

Reviewer 2: I find the comma fine, but it doesn't have to be there.

Reviewer: It's an awkward construction, yea. Just like all appends, in my opinion. Main alternative would be to shift around the word order, though I'm not sure that attaching it to the previous sentence like this works here either. It also doesn't work to just start the sentence with it, because there's nothing to contrast it against that way.

Just snipping off "though" may be the least bad option?

Author: ```suggestion
			`	With that he steps away directly to meet an aide who had been hurrying towards you both, intent on ignoring the bubble. From across the room you catch Xilin with his eyebrows raised, giving you a curious look despite being engaged in his own conversation. Neither of them speak to you again for the remainder of the meetings.`
```

			label publicbroadcast

> 			`	Eventually you follow the diplomats to a room called the grand Borealis Chamber, "Where Mountains Touch The Sky," as Teeneep makes the first publicly broadcast speech from the Hai to humanity. The three heads of state announce a resolution granting safe passage to all Hai in human space, as long as they register with customs on Mountaintop before entering, and Teeneep expresses similar sentiments for humans visiting Hai space. It is announced that extensive surveillance fleets will be posted on both sides of the wormhole to ensure safety. This meeting will now turn into a multi-day summit to discuss the creation of a formal treaty.`

Reviewer: ```suggestion
			`	Eventually you follow the diplomats to a room called the Grand Borealis Chamber, "Where Mountains Touch The Sky," as Teeneep makes the first publicly broadcast speech from the Hai to humanity. The three heads of state announce a resolution granting safe passage to all Hai in human space, as long as they register with customs on Mountaintop before entering, and Teeneep expresses similar sentiments for humans visiting Hai space. It is announced that extensive surveillance fleets will be posted on both sides of the wormhole to ensure safety. This meeting will now turn into a multi-day summit to discuss the creation of a formal treaty.`
```

> 			`	Eventually you follow the diplomats to a room called the grand Borealis Chamber, "Where Mountains Touch The Sky," as Teeneep makes the first publicly broadcast speech from the Hai to humanity. The three heads of state announce a resolution granting safe passage to all Hai in human space, as long as they register with customs on Mountaintop before entering, and Teeneep expresses similar sentiments for humans visiting Hai space. It is announced that extensive surveillance fleets will be posted on both sides of the wormhole to ensure safety. This meeting will now turn into a multi-day summit to discuss the creation of a formal treaty.`

Reviewer: ```suggestion
			`	Eventually you follow the diplomats to a grandiose room called the Borealis Chamber, "Where Mountains Touch The Sky," as Teeneep makes the first publicly broadcast speech from the Hai to humanity. The three heads of state announce a resolution granting safe passage to all Hai in human space, as long as they register with customs on Mountaintop before entering, and Teeneep expresses similar sentiments for humans visiting Hai space. It is announced that extensive surveillance fleets will be posted on both sides of the wormhole to ensure safety. This meeting will now turn into a multi-day summit to discuss the creation of a formal treaty.`
```
Alternatively
